test/functional/page_objects/common_page.ts

```diff
@@ -502,11 +503,30 @@ export class CommonPageObject extends FtrService {
     }
   }
 
+  formatTime(time: TimeStrings, fmt: string = 'MMM D, YYYY @ HH:mm:ss.SSS') {
+    return Object.keys(time)
+      .map((x) => moment(time[x], [fmt]).format())
+      .reduce(
+        (acc, curr, idx) => {
+          if (idx === 0) acc.from = curr;
+          acc.to = curr;
+          return acc;
+        },
+        { from: '', to: '' }
+      );
+  }
```

Comment:
@wayneseymour could add description to this function? I think it is worth to mention the reason, should help people understand how things work and why need to it this way.

Reply:
@dmlemeshko that's a great idea mate.